src/cryptoadvance/specter/wallet.py

```diff
@@ -2073,3 +2075,44 @@ def addresses_info(
 
     def __repr__(self) -> str:
         return f"<{self.__class__.__name__} name={self.name } alias={self.alias}>"
+
+    def convert_rawtransaction_to_psbt(self, rawtransaction) -> str:
```

Comment:
Can you migrate that method maybe to `util/tx.py` ? There is not much `self` in that method apart from self.rpc and maybe you pass the rpc-object into the method?
wallet.py is already way too long. We need to find ways how to make it even shorter than it is now.